### DISCUSSION TITLE
Min rule not working properly on array

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#287] [Questions] [answered]
The rule 'array|min:1' does not seem to work or update properly. I even tried with a custom rule:

```
const minRows = class extends Validator {
  get minRows () {
    return this.attributes[0];
  }

  get minRowsmsg () {
    return this.attributes[1];
  }

  get msg() {
    return this.minRowsmsg? this.minRowsmsg : `At least ${this.minRows} entries required`
  }

  check(value: any) {
    return value.length >= this.minRows;
  }
}
```

When I validate the form with an empty array, the validation error correctly appears, but it does not clear when adding a entry.
If I add a row and then validate after, it again validates correctly with no error, but when removing the row the error does not appear.
It seems like the validation does not update, perhaps because it does not detect a change in the array? Any idea on how to solve this? 

https://stackblitz.com/edit/github-nzqvai-2rjq4g?file=src%2FApp.vue,src%2Fstyle.css,vueform.config.ts,src%2Fmain.js

**Comment by adamberecz** [comment, marked as answer]
The issue is with the the way you're setting the value. The `this.value.push` will not trigger the reactivity required for revalidation:
```js
    addNewItem() {
      this.value = [...this.value, {
        title: 'My Title',
        organization: 'My Org',
        position: 'My position',
        contactpreferredname: 'My contact',
        scontacttitle: 'My Contact Title',
        contactEmailAddress: 'My contact email',
      }]
    },
```
https://stackblitz.com/edit/github-nzqvai-tvqefm
